hash_practice/exercises.py

```diff
@@ -2,18 +2,46 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n^2)? The fucntion only loops through the length of strings for 
+                                the number of letters in each string?
+        Space Complexity: O(n)? Theres only 2 varaibles being stored...sortedWord and anagramsHash?

```

Comment:
🪐 Yes! `anagramsHash` can have at most n elements where n is the length of `strings`. Overall the combined length of the values will also be n elements.